Engine/src/Engine/SceneBuilder/Scene.h

```diff
@@ -24,6 +28,12 @@ namespace Engine
     public:
 
         #pragma region Entity Management
+
+        unsigned int m_fbo;
+        unsigned int textureColorbuffer;
```

Comment:
'm_' for style guidelines. For textureColorbuffer, and quadVAO/quadVBO.

Reply:
line has been removed from this PR